include/motor-control/core/motion_controller.hpp

```diff
@@ -71,15 +51,15 @@ class MotionController {
         queue.try_write(msg);
     }
 
-    void enable_motor() { set_pin(hardware_config.enable); }
+    void stop() { hardware.stop_timer_interrupt(); }
```

Comment:
When we talked in pairing session you suggested that this is not the right way to stop. Should we leave a TODO or ticket this?